README.md

```diff
@@ -21,15 +21,18 @@ If you've created an example with GDevelop, you can submit it to be shared with
 
 1. **Create your game** with GDevelop.
 2. Make sure to follow the [requirements and best practices on this page](http://wiki.compilgames.net/doku.php/gdevelop5/community/guide-for-submitting-an-example).
-  > Note that for now, we're trying to keep a fairly high quality bar for examples, so you might be asked to adapt your game according to reviewer feedbacks. Don't feel bad about this! This is normal process and here to help making examples as good as possible for new users. If we take too much time to review your example, you can send a ping on the issue. ⏰
+  > Note that for now, we're trying to keep a fairly high quality bar for examples, so you might be asked to adapt your game according to reviewer feedbacks.
+  > Don't feel bad about this! This is normal process and here to help making examples as good as possible for new users.
+  > If we take too much time to review your example, you can send a ping on the issue. ⏰
```

Comment:
I just noticed that this change didn't result in a visible change. 
What was your goal with this change?  Were you trying to make a bulleted list?

Reply:
Changed to make it easy to understand the content when reading as a file!

Reply:
That makes sense, thanks for explaining!  :)